ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -805,6 +885,17 @@ private MockId getUniqueId() {
     return ids;
   }
 
+  private MockId putSingleBlob(String key, short accountId, short containerId) throws StoreException {
```

Comment:
is it possible to modify the `put()` function in this class instead of writing a new one? If you use that function, you can even check the `storeInfo` that gets returned on GET

Reply:
The problem here is we want specify accoundId and containerId. We need to add two arguments for put().